docs/GLOSSARY.md

```diff
@@ -10,26 +14,50 @@ Examples:
 - Phazon Mining Tunnel in Metroid Prime
 - Big Pink in Super Metroid
 
+## Burger King / BK
+
+A strictly informal term used to describe the situation of being unable to progress in a [Multiworld](#multiworld) [Session](#session). This occurs when a player has exhausted all readily available [Checks](#item-check) and must wait for another player to send them [Progression](#progression) before they can continue. The term was coined after a player in this state left to get food from the eponymous restaurant and had yet to obtain Progression by the time they returned.
```

Comment:
There are two types: "Logical BK", where all checks within the generator's ability are exhausted and "Full BK" which is only limited by the player's ability.

Reply:
Alright, added that distinction.